docs/source/conf.py

```diff
@@ -208,11 +208,3 @@ def find_version(*file_paths):
         "Miscellaneous",
     )
 ]
-
-
-# Taken from https://github.com/pyro-ppl/pyro/blob/dev/docs/source/conf.py#L213
-# @jpchen's hack to get rtd builder to install latest pytorch
-# See similar line in the install section of .travis.yml
-if "READTHEDOCS" in os.environ:
-    os.system("pip install torch==1.10.0+cpu -f https://download.pytorch.org/whl/torch_stable.html")
-    os.system("pip install pyro-ppl")
```

Comment:
I'm hoping that adding `requirements.txt` to `.readthedocs.yaml` makes this obsolete.

Reply:
We will still need to install pyro somewhere.